front/src/App.js

```diff
@@ -28,15 +31,47 @@ class App extends React.Component {
             )
     }
 
+    handleresearchIsVisible() {
+        this.setState({ researchIsVisible: !this.state.researchIsVisible })
+    }
+
+    handleSelectedTags(instrument) {
+        this.state.selectedTags.includes(instrument)
+            ? this.setState({
+                  selectedTags: this.state.selectedTags.filter(
+                      tag => tag !== instrument
+                  )
+              })
+            : this.setState({
+                  selectedTags: [...this.state.selectedTags, instrument]
+              })
+    }
+    render() {
+        return (
+            <div className='App'>
+                {this.state.researchIsVisible ? null : (
```

Comment:
use a more suited operator ;)

Reply:
resolved with 
{this.state.researchIsVisible || (